python/paddle/distributed/auto_parallel/api.py

```diff
@@ -667,11 +729,8 @@ def to_static(
             pipeline parallelism). Default: None.
 
     Returns:
-        DistModel: A DistModel tha contains corresponding computational graph
-            for the input ``layer`` and provides APIs for training, evaluation
-            and prediction.
-        DistributedDataLoader: An optimized data loader that can be used
-            to generate data.
+        DistModel: A ``DistModel`` instance converted the input ``layer``.
+        DistributedDataLoader: An optimized data loader that can be used to generate data.
```

Comment:
It should be pointed that `DistributedDataLoader` has the same usage with `DataLoader` ?

Reply:
Done